src/apps/chifra/internal/when/README.md

```diff
@@ -34,6 +34,7 @@ Flags:
   -x, --fmt string   export format, one of [none|json*|txt|csv]
   -v, --verbose      enable verbose (increase detail with --log_level)
   -h, --help         display this help screen
+  -o, --cache        force the results of the query into the cache
```

Comment:
Sorry to be such a total pain in the ass, but please move this up three line to before the `--fmt` line and change the text to `for some commands, force the results of the query into the cache.

I always like to see `--help` last in the command list.